manifests/plugin.pp

```diff
@@ -46,6 +46,7 @@
     timeout => $timeout,
     cwd     => $destination,
     require => Exec["rbenv::plugin::checkout ${user} ${plugin_name}"],
+    onlyif  => 'git remote update; if [ "$(git rev-parse @{0})" = "$(git rev-parse @{u})" ]; then return 0; else return 1; fi ]',
```

Comment:
is there a reason why we don't use [vcsrepo](https://github.com/puppetlabs/puppetlabs-vcsrepo) here?